src/Uno.UI/UI/Xaml/XamlScope.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Text;
+using Uno.UI.DataBinding;
+
+namespace Windows.UI.Xaml
+{
+	/// <summary>
+	/// This is a helper to support resolution of StaticResource/ThemeResource references using the 'Xaml-tree scope' similar to UWP, as
+	/// opposed to the visual tree. In particular it allows correct resolution during template resolution, where the visual tree may be
+	/// arbitrarily distant from the xaml tree.
+	/// </summary>
+	internal class XamlScope
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)

Reply:
It's `internal`, robo-Carl